pkg/notifiers/format.go

```diff
@@ -91,7 +91,7 @@ const bplPolicyFormat = `
 	{{stringify "Type:" .GetResource.ResourceType | list}}
 	{{stringify "Cluster:" .GetResource.ClusterName | list}}
 	{{stringify "ClusterId:" .GetResource.ClusterId | list}}
-	{{stringify "Namespace:" .GetResource.Namespace | list}}
+	{{if .GetResource.Namespace }}{{stringify "Namespace:" .GetResource.Namespace | list}}{{end}}
```

Comment:
Note to self. Once https://github.com/stackrox/stackrox/pull/7725 is merged, rebase and update format_test.go to test for this case as well